resources/assets/js/components/GroupEvents.vue

```diff
@@ -0,0 +1,291 @@
+<template>
+  <div>
+    <CollapsibleSection class="lineheight d-none d-md-block" collapsed :count="upcoming.length" count-badge :heading-level="headingLevel">
+      <template slot="title">
+        <div class="d-flex justify-content-between w-100">
+          <div>
+            {{ translatedTitle }}
+            <b-btn v-if="calendarCopyUrl" class="ml-2" variant="primary" @click="showCalendar">
+              <b-img-lazy src="/images/subs_cal_ico.svg" />
+            </b-btn>
+          </div>
+          <b-btn variant="primary" href="/party/create" class="align-self-center">
+            <span class="d-none d-md-block">
+              {{ translatedAddEvent }}
+            </span>
+          </b-btn>
+        </div>
+      </template>
+      <template slot="content">
+        <b-tabs class="ourtabs w-100">
+          <b-tab active title-item-class="w-50" class="pt-2">
+            <template slot="title">
+              <div class="d-flex justify-content-between">
+                <div>
+                  <b>{{ translatedUpcoming }}</b> ({{ upcoming.length }})
+                </div>
+              </div>
+            </template>
+            <p v-if="!upcoming.length">
+              {{ translatedNoUpcoming }}.
+            </p>
+            <b-table-simple v-else sticky-header="50vh" responsive class="pl-0 pl-md-3 pr-0 pr-md-3 pb-2 mb-2" table-class="m-0 leave-tables-alone">
+              <GroupEventsTableHeading />
+              <b-tbody class="table-height">
+                <GroupEventSummary v-for="e in upcomingToShow" :key="'event-' + e.idevents" :idevents="e.idevents" :canedit="canedit" />
+              </b-tbody>
+            </b-table-simple>
+            <div class="text-center" v-if="limit">
```

Comment:
I guess `limit` relates to my previous question about why we removed the `take` from the backend query.